dll/win32/shell32/brfolder.cpp

```diff
@@ -1353,16 +1361,22 @@ SHBrowseForFolderW(LPBROWSEINFOW lpbi)
 {
     TRACE("%p\n", lpbi);
 
+    COleInit OleInit; // TODO: MSDN says the caller must initialize COM so why are we doing it?
```

Comment:
Most likely because some callers are fucking up

Reply:
You are probably correct. Too risky to change it now, I will update the comment to say why we are doing it.